lib/active_model/serializer/adapter/json_api.rb

```diff
@@ -21,11 +48,16 @@ def initialize(serializer, options = {})
         def serializable_hash(options = nil)
           options ||= {}
 
-          if serializer.respond_to?(:each)
-            serializable_hash_for_collection(options)
-          else
-            serializable_hash_for_single_resource(options)
-          end
+          hash =
+            if serializer.respond_to?(:each)
+              serializable_hash_for_collection(options)
+            else
+              serializable_hash_for_single_resource(options)
+            end
+
+          ApiObjects::JsonApi.include_member? && hash.merge!(ApiObjects::JsonApi.member)
```

Comment:
I do, esp. if you're ok with it... I'm willing to be persuaded... the whole ApiObjects::JsonApi is just an idea I'm playing with.. not very useful if it's the only one, but might lead to good abstractions for other objects, like errors, paginations, meta, links, etc.

What do you think of renaming `member` to `object`?  It would align with serializer.object and make sense in context.